server/app/services/ti/TrustedIntermediaryService.java

```diff
@@ -279,15 +279,28 @@ private ImmutableList<AccountModel> searchAccounts(
     return allAccounts.stream()
         .filter(
             account ->
-                ((account.newestApplicant().get().getApplicantData().getDateOfBirth().isPresent()
+                // TODO (#5503): Remove checking at WKP when removing feature flag
+                (((account.newestApplicant().get().getDateOfBirth().isPresent()
+                            || account
+                                .newestApplicant()
+                                .get()
+                                .getApplicantData()
+                                .getDateOfBirthAtWellKnownPath()
+                                .isPresent())
                         && maybeDOB.isPresent()
-                        && account
-                            .newestApplicant()
-                            .get()
-                            .getApplicantData()
-                            .getDateOfBirth()
-                            .get()
-                            .equals(maybeDOB.get()))
+                        && (account
+                                .newestApplicant()
+                                .get()
+                                .getDateOfBirth()
+                                .get()
+                                .equals(maybeDOB.get())
+                            || account
+                                .newestApplicant()
+                                .get()
+                                .getApplicantData()
+                                .getDateOfBirthAtWellKnownPath()
+                                .get()
+                                .equals(maybeDOB.get())))
                     || (!missingParams.contains(SearchParameters.ParamTypes.NAME)
                         && account
                             .getApplicantName()
```

Comment:
suggestion (maybe blocking): I think `account.getApplicantName` calls out to `applicantData.getApplicant` name so you might need to handle both cases in `account.getApplicantName` as well.

Reply:
TI client names show up now!